decrypt_test.go

```diff
@@ -6,18 +6,33 @@ import (
 )
 
 func TestDecrypt(t *testing.T) {
-	fixture := UnmarshalTestFixture(EncryptedTestFixture)
-	p7, err := Parse(fixture.Input)
-	if err != nil {
-		t.Fatal(err)
+	tests := []struct {
+		name      string
+		algorithm string
+		hashFunc  string
+		fixture   string
+		expected  []byte
```

Comment:
perhaps `expected` should be a string. there's too much `[]byte("")` going on in the declaration of the structs. You can simply cast to `[]byte` once when asserting.

Reply:
Fixed in b1fb0e54346f2f8b218ca002427fe0eff22564f9